src/versioning.ts

```diff
@@ -82,27 +82,28 @@ export default function determineTargetsFromConfig(
 }
 
 /**
- * Take a list of targets returned from browserslist api, return the lowest version
- * version of each target
+ * @param targetslist - List of targest from browserslist api
+ * @returns  - The lowest version version of each target
+ *
  */
 export function versioning(targetslist: Array<string>): Array<TargetListItem> {
   return (
-    targetslist
-      // Sort the targets by target name and then version number in ascending order
-      .map((e: string): TargetListItem => {
-        const [target, version] = e.split(" ");
-        return {
-          target,
-          version,
-          parsedVersion:
-            version === "all"
-              ? 0
-              : version.includes("-")
-              ? parseFloat(version.split("-")[0])
-              : parseFloat(version)
-        };
-      })
-      // Sort the targets by target name and then version number in descending order
+    targetslist // Sort the targets by target name and then version number in ascending order
```

Comment:
nit: adding adjacent comments prevents prettier from new line formatting and other kinds of formatting. Best to keep this comment on the line above it.